packages/spor-icon-react/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "files": [
     "svg",
-    "react"
+    "dist"
```

Comment:
This fixes a bug where the built icon files weren't included in the npm package. Awkward!